pages/XSS_in_Converting_File_Content_to_Text

```diff
@@ -0,0 +1,27 @@
+---
+
+layout: col-sidebar
+title: XSS in Converting File Content to Text
+author: Mohammad Reza Omrani
+contributors: 
+permalink: /attacks/XSS_in_Converting_File_Content_to_Text
+tags: [attack, XSS]
+
+---
+
+ {% include writers.html %}
+
+## Description
+
+Attackers can execute JavaScript by converting the content
+of a file to text, which is commonly known as Cross-Site Scripting (XSS).
+If an image containing XSS payload is imported into an image-to-text program,
+it can execute JavaScript code. This vulnerability has been verified by testing
+some services that translate text from photos and convert photos to text.
+This same process may apply to other vulnerabilities as well!
+## Examples
+
+Attackers can use programs like Paint to write payloads on blank white photos and send them to targets.
+
+# This photo must be included:
+# https://drive.google.com/file/d/1Cq37yEwSz1zhnrEZMBnq81mLRJwEiT8L/view?usp=sharing
```

Comment:
Then include it??

Reply:
What do you need to know?

Attach the file to the PR by adding it to the appropriate part of assets/images/ if you need the syntax for inserting images just search the repo for other examples (not, gif, jpg...). I'm on my phone right now so can't be more specific. (It might be easier to do via dev.GitHub.com 🤷‍♂️)